src/toDo.test.js

```diff
@@ -0,0 +1,15 @@
+const LocalStorage = require('./storage.js');
+const ToDoList = require('./toDo.js');
```

Comment:
Well done using the class model to organize your code base :+1: 
One thing to mention here :point_down: 

While we added babel plugin, to transform ES6 syntax to commanJS as shown here
> https://github.com/Buyaki01/to_do_list/blob/407f69efa02868161ad17044ce2a9408d029f7a7/package.json#L10

Why we don't use `import/export` as known as one of the Js best practice to be used :thinking:  

[OPTIONAL] Change the syntax to use `import/export`

Reply:
Thank you for your valuable feedback. We will implement the suggested changes to our project